dll/win32/version/version.c

```diff
@@ -979,6 +979,14 @@ static BOOL VersionInfo32_QueryValue( const VS_VERSION_INFO_STRUCT32 *info, LPCW
 
     /* Return value */
     *lplpBuffer = VersionInfo32_Value( info );
+
+#ifdef __REACTOS__
+    /* If the wValueLength is zero, then set a UNICODE_NULL only return string.
+     * Use the NULL terminator from the key string for that. This is what Windows does, too. */
+    if (!info->wValueLength)
+      *lplpBuffer = (PVOID)(info->szKey + wcslen(info->szKey));
+#endif
+
```

Comment:
Question: When replacing in the unpatched ros the version.dll with the version.dll from 2k3sp2... Does that fix the issue too? (I am trying to find out whether you do touch the correct module here with your fix)

Reply:
Yes, replacing the ReactOS version.dll with the one from W2K3SP2 does fix the issue.

Reply:
![version_dll_w2k3wp2](https://github.com/user-attachments/assets/a12d45ed-3062-45c1-bd7d-895bf1602d57)

Reply:
Thank you, I do recommend to merge this PR after https://github.com/reactos/reactos/pull/7439 . Resolved.